### PR TITLE
Add basic documentation setup for Reepay

### PR DESCRIPTION
**Pull request by bjarnef**
Just basic documentation for Reepay. Pretty much a copy of the empty DIBS documentation structure.

### CODE REVIEW

File: content/docs/payment-providers/reepay/changelog.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+<changelog>
+<changelog-group category="Added">  
+
+    
+* Added Reepay Payment Provider
+
+
+</changelog-group>
+</changelog>
```
Comment (mattbrailsford) [inline content/docs/payment-providers/reepay/changelog.md:19]: Don't need this for the initial commit

File: content/docs/payment-providers/reepay/package.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+id: Vendr.PaymentProviders.Reepay
+name: Reepay
+logo: reepay.png
```
Comment (mattbrailsford) [inline content/docs/payment-providers/reepay/package.yml:3]: Missing image is breaking the preview build. Image needs adding to`src\assets\images\logos`

**Comment by bjarnef** [comment]
I have updated this.
For reference Reepay has their logo available here: https://reepay.com/asset-library/

**Comment by mattbrailsford** [comment]
Ok, Vale is currently failing due to Reepay being a word it doesn't know about. You might need to add it to this list https://github.com/vendrhub/vendr-documentation/blob/master/.github/styles/vocab.txt

Also, you'll want to add the vale off / on comments I've recently added to the versions.md files, and example you can find here https://raw.githubusercontent.com/vendrhub/vendr-documentation/master/content/docs/core/versions.md

File: content/docs/payment-providers/reepay/1.0.0/sub-packages.yml [hunk 3]
```diff
@@ -0,0 +1,2 @@
+- name: Reepay
+  alias: reepay
```
Comment (mattbrailsford) [inline content/docs/payment-providers/reepay/1.0.0/sub-packages.yml:2]: Similar to QuickPay, this isn't matching the folder name on disk

**Pull request closed without merging**